aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -55,12 +59,18 @@ The `Movie` class contains:
   * Only the date is displayed, not time information.
 * The question mark after `string` indicates that the property is nullable. For more information, see [Nullable reference types](/dotnet/csharp/nullable-references).
 
+[DataAnnotations](xref:System.ComponentModel.DataAnnotations) are covered in a later tutorial.
+
 <a name="dc7"></a>
 
 ### Add NuGet packages and EF tools
 
 [!INCLUDE[](~/includes/add-EF-NuGet-SQLite-CLI-7.md)]
 
+In Visual Studio Code, from the *Run* menu, select *Start Debugging* or press <kbd>F5</kbd> to build and run the the project with debugging. 
+
+In the *Panel* below the editor region, select the *PROBLEMS* tab, or from the *View* menu, select *Problems* if it is not currently in view. Verify there are no compilation errors.
```

Comment:
If this is just to check for compile errors, why is debugging needed?

Reply:
You are right, I should have set it to without debugging.  Fixing..